docs/5.0/preview/cloud.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Teleport Cloud Preview
+description: Teleport hosted and managed by Teleport team.
+---
+
+# Teleport Cloud Preview
+
+Teleport cloud is a managed an hosted service of Teleport.
+Request access to preview [here](https://goteleport.com/get-started/).
+
+## Status
+
+Teleport cloud is currently only recommended for environments that do not require
+meeting strict compliance standards or SLA uptime guarantees.
+
+We believe that Teleport cloud environment is secure, however we scrutinizing
+and executing on our security roadmap and working with independent security
+auditors to identify any gaps.
+
+## Roadmap
+
+For the first half of 2021, we will be working with auditors to realize the
+threat modeling and security controls.
+
+Once we realize, test and audit the security model we will start working towards meeting
+compliance programs such as FedRamp.
+
+## Billing
+
+Usage will be collected, but billing will not be set up until February.
+Preview version will be free of charge to preview customers until
+we commit to SLA in February.
+
+## Security
+
+Each customer Teleport is running as separate deployment
+in a multi-tenant Kubernetes cluster on the AWS infrastructure.
+
+Each deployment is using at rest encryption for customer
+data including session recordings and user records.
+
+Password hashes are generated using
+[Golang's bcrypt](https://pkg.go.dev/golang.org/x/crypto/bcrypt).
+
+Teleport uses [letsencrypt.org](https://letsencrypt.org/) to issue
+certificates for every customer.
+
+## Managed Teleport Settings
+
+SSH sessions are recorded [on nodes](/architecture/nodes/#session-recording).
+
+Teleport does not terminate SSH sessions when using OpenSSH and `tsh` sessions.
+
+It terminates Application, Database and Kubernetes session on the proxy.
+
+## Data retention
+
+The session records are stored in S3 using at rest encryption and
+are not expired and retention policies are not set.
+This is subject to change after GA release.
+
+Customer data including session events is backed up using DynamodDB
+point in time backup system with recovery option up to 35 days
+and is not configurable.
+
+## High availability
+
+Clusters are deployed in a single AWS region in 2 availability zones.
+AWS guarantees [99.99%](https://aws.amazon.com/compute/sla/) of monthly uptime percentage.
+
+Teleport cloud preview version does not commit to any SLA. It sets availability
+objective to 99.7% of monthly uptime percentage or maximum of two hours of downtime per month.
+
+We plan to get to 99.99% uptime SLA in February.
```

Comment:
I don't see how this is possible.

Reply:
ok, removed specific date, but we'd have to commit to some SLA in Feb. We can discuss offline